ru/index.rst

```diff
@@ -14,15 +14,15 @@ Welcome
 
 .. container:: offline-download
 
-    **Read the Book Anywhere**
+    **Читайте руководство где угодно**
 
     .. image:: /_static/img/read-the-book.jpg
 
-    Enjoy the CakePHP cookbook almost anywhere. Available as both a PDF and
-    EPUB, you can now read it on more devices, as well as offline.
+    Наслаждайтесь книгой рецептов CakePHP почти везде. Она дступна как в формате PDF,
+    так и в EPUB, вы можете теперь читать ее на многих устройствах, даже офлайн.
 
-    - `PDF <../_downloads/en/CakePHPCookbook.pdf>`_
-    - `EPUB <../_downloads/en/CakePHPCookbook.epub>`_
+    - `PDF <../_downloads/ru/CakePHPCookbook.pdf>`_
```

Comment:
@markstory maybe you will remember this better than I do but does the pdf generation is working with Russian alphabet ?